### PR TITLE
refactor: remove cancelling from slashing

### PR DESCRIPTION
**Pull request by 8sunyuan**
Removing cancelling from Slashing request lifecycle but big TODO is to implement this at the AVS middleware contracts level

### CODE REVIEW

File: src/contracts/core/DelegationManager.sol [hunk 1]
```diff
@@ -572,10 +563,6 @@ contract DelegationManager is Initializable, OwnableUpgradeable, Pausable, Deleg
             );
         }
 
-        uint32 epochForEndOfSlashability = EpochUtils.getEndOfSlashabilityEpoch(withdrawalCreationEpoch(withdrawalRoot));
```
Comment (gpsanant) [inline src/contracts/core/DelegationManager.sol:575]: we still need this to only factor in slashing from up until the epoch after they queued the withdrawal

Comment (8sunyuan) [reply src/contracts/core/DelegationManager.sol:575]: Are you not still subject to slashing while queued withdrawing?

Comment (gpsanant) [reply src/contracts/core/DelegationManager.sol:575]: a withdrawal queued in epoch `n` should be subject to slashing until the end of epoch `n+1` and no longer

Comment (8sunyuan) [reply src/contracts/core/DelegationManager.sol:575]: Right because the withdrawal can be completed whenever and not necessarily in epoch n + 2

Comment (8sunyuan) [reply src/contracts/core/DelegationManager.sol:575]: https://github.com/Layr-Labs/eigenlayer-contracts/pull/619/commits/0e3dd0843b36d4e3847e0ddbbd8ff8cdb633439a

**Comment by gpsanant** [comment]
@8sunyuan can this be closed?

**Pull request closed without merging**